invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/requests.js

```diff
@@ -323,6 +324,55 @@ export const RequestsFacets = ({ aggs, currentResultsState }) => {
   );
 };
 
+export const RDMRequestsEmptyResults = (props) => {
+  const { queryString, userSelectionFilters } = props;
+  const is_open = userSelectionFilters.some(
+    (obj) => obj.includes("is_open") && obj.includes("true")
+  );
+  const filtersToNotReset = userSelectionFilters.find((obj) =>
+    obj.includes("is_open")
+  );
+  const elementsToReset = {
+    queryString: "",
+    page: 1,
+    filters: [filtersToNotReset],
+  };
+  return (
+    <>
+      <Segment placeholder textAlign="center">
+        {is_open && !queryString ? (
+          <>
+            <Header as="h2" icon>
+              <span>{i18next.t("All done!")}</span>
+            </Header>
+            <span className="grey-color">
```

Comment:
this should be solved with subheader component of semantic ui, instead of span and custom CSS class. Custom CSS should be the last resort

Reply:
Thanks! Was not aware of the subheader cmp, will try to explore more semantic ui react next times before going for custom css